packages/bundler/src/modules/BundleManager.ts

```diff
@@ -190,6 +197,18 @@ export class BundleManager {
         this.mempoolManager.removeUserOp(entry.userOp)
         continue
       }
+
+      for (const storageAddress of Object.keys(validationResult.storageMap)) {
+        if (
+          storageAddress.toLowerCase() !== entry.userOp.sender.toLowerCase() &&
+          knownSenders.includes(storageAddress.toLowerCase())
+        ) {
+          console.log('UserOperation read a storage of a different sender entity')
```

Comment:
don't leave comments, especially not one with no context.
e.g. can be debug( userop of ${entry.userop.sender} references  another sender ${storageAddress}

(also, missing a test)

Reply:
on the bundler repo, I'd expect a at least a minimal test. on the spec-test there should be an exhaustive list of tests.

Reply:
> on the bundler repo, I'd expect a at least a minimal test. on the spec-test there should be an exhaustive list of tests.

There are barely any tests in a bundler repo. Also the tests appear to run against `hardhat` node, I don't have any way of testing this code without `go-ethereum` as far as I know. There is a corresponding test in the spec-test repo.

Reply:
please see the ValidationManager.test.ts
it contains a set of tests (not comprehensive, but still) for all basic rules.
(and yes, it requires geth)

Reply:
Pushed a test for this specific rule.